README.md

```diff
@@ -2,6 +2,26 @@
 
 A script to search in HMAP (http://hmap.fr/) set of music scores. Basically, a PHP script to search in a CSV.
 
+## Document access token
+
+Tokens are used to regulate access to scores. Nobody can access to scores without it, and it has an expiration date.
+
+If the used link to the scores is used for malicious purpose, it can be quickly deactivated to stop access from the leaked link. Then a new one can be created to provide a new access to target users.
+
+To create a new token, create a file in repository `private/tokens/` in the format `YYYYMMDD-KEY.txt`, by replacing `YYYYMMDD` by a given date (in format year-month-day) which is the limit of activation of the token (passed this date, the token is disabled) and `KEY` by any alphanumeric string of characters (preferably randomly generated).
```

Comment:
```suggestion
To create a new token, create a file in repository `private/tokens/` in the format `YYYYMMDD-KEY.txt`, by replacing `YYYYMMDD` by a given date (in format year-month-day) which is the date after which the token is disabled, and `KEY` by any alphanumeric string of characters. `KEY` *must be* randomly generated, because anyone that knows can access all scores. All this process can be handled by `/admin/token.php`. This script *MUST* be protected from public access.
```